addons/io_scene_gltf2/blender/com/gltf2_blender_material_helpers.py

```diff
@@ -20,7 +20,30 @@ def get_gltf_node_name():
 def create_settings_group(name):
     gltf_node_group = bpy.data.node_groups.new(name, 'ShaderNodeTree')
     gltf_node_group.inputs.new("NodeSocketFloat", "Occlusion")
+    thicknessFactor  = gltf_node_group.inputs.new("NodeSocketFloat", "Thickness")
+    thicknessFactor.default_value = 1.0
     gltf_node_group.nodes.new('NodeGroupOutput')
     gltf_node_group_input = gltf_node_group.nodes.new('NodeGroupInput')
     gltf_node_group_input.location = -200, 0
-    return gltf_node_group
\ No newline at end of file
+    return gltf_node_group
+
+def get_gltf_pbr_non_converted_name():
+    return "glTF Non Converted BPR Extensions"
```

Comment:
There's a typo here:  BPR -> PBR.  Also I'm not sure "Non Converted" is the right terminology, perhaps we'll talk about this at Monday's meeting and someone will have a better idea for the name.

Reply:
Fixed! Of course, we can discuss how to name it better